docs/docusaurus.config.ts

```diff
@@ -0,0 +1,126 @@
+import type * as Preset from "@docusaurus/preset-classic";
+import type { Config } from "@docusaurus/types";
+import { themes as prismThemes } from "prism-react-renderer";
+
+const config: Config = {
+  title: "OpenDevin",
+  tagline: "Code Less, Make More",
+  favicon: "img/logo.png",
+
+  // Set the production url of your site here
+  url: "https://OpenDevin.github.io",
+  baseUrl: "/OpenDevin/",
+
+  // GitHub pages deployment config.
+  organizationName: "OpenDevin",
+  projectName: "OpenDevin",
+  trailingSlash: false,
+
+  onBrokenLinks: "throw",
+  onBrokenMarkdownLinks: "warn",
+
+  // Even if you don't use internationalization, you can use this field to set
+  // useful metadata like html lang. For example, if your site is Chinese, you
+  // may want to replace "en" with "zh-Hans".
+  i18n: {
```

Comment:
Are we losing README-zh entirely?

Reply:
Well, we only have the main readme in Chinese anyway so we could keep that if we wanted to. In addition, Docusaurus has great i18n support.

However, I don't know how feasible it is at this point to translate everything and, more importantly, keep everything in sync between translations if things change. I'm not super convinced that we're at a point where we should spend time on multilingual docs. Creating them is one thing and can be done by volunteers but every time somebody update anything things basically break. If I change something I'd be able to change the docs for German and English, maybe Spanish that's it. Similar probably for most other people. How would we maintain consistency?